src/main/java/me/botsko/prism/database/mysql/SelectQueryBuilder.java

```diff
@@ -330,7 +330,7 @@ protected String group() {
     @Override
     protected String order() {
         final String sort_dir = parameters.getSortDirection();
-        return " ORDER BY " + tableNameData + ".epoch " + sort_dir + ", x ASC, z ASC, y ASC, id " + sort_dir;
+        return " ORDER BY " + tableNameData + ".epoch " + sort_dir + ", id " + sort_dir;
```

Comment:
Have you fully tested these changes? We're sorting by coordinate, most importantly y, so we don't put a falling block above blocks which haven't yet been put back. Same with attached blocks like torches and such. There's no guarantee that records will be recorded in the right order.